### DISCUSSION TITLE
World buffs unabled for non-current content.

### DISCUSSION THREAD

**Discussion by Eccies** [chromiecraft/chromiecraft#3380] [Suggestions]
Hello,

Since BWL is out, what do you guys think of enabling world buffs for previous content, aka MC and Ony for now ? 

Leave current without world buffs for the challenge ofc. 

Molten core and Onyxia are cleared by the majority of the guilds without any difficulty. I think adding world buffs to those raids won't affect people's progress and adds some fun to people who wants to see big numbers. 

Thank you for reading ! =)

**Comment by KaneoHunter** [comment]
I like this, it would help to make the world more alive, with people going around collecting world buffs.

Maybe we can have a Ony/Nef automatically dropping on a 3 hour cycle, since I think it will be hard to get consistent head drops with our population?

This one is more to my taste, but maybe make pvp flagging a requirement for world buffs? There's been an outcry for more world pvp for a long time, and I feel that world buffing becomes an arbitrary chore in a pve setting, instead of a game to collect them all and get them to a raid.
